runtime/queries/metricsview.go

```diff
@@ -258,3 +258,52 @@ func repeatString(val string, n int) []string {
 	}
 	return res
 }
+
+func validateAndGetDimension(mv *runtimev1.MetricsView, dimName string) (string, error) {
```

Comment:
Call it `metricsViewDimensionToColumn` or something like that

Reply:
Also, apply `safeName` to the result so the callers don't need a separate statement for it

Reply:
And the places it's called, I guess the output vars should be `colName` or something instead of `dimName`

Reply:
Renaming to `dimensionToSafeColumn`. Since already in metricsview package dont think it makes sense to add it to the name.

Reply:
My bad. The package is not `metricsview`. Will rename appropriately.